src/main/resources/requests/subscriptionRequests.ts

```diff
@@ -15,4 +17,20 @@ export class SubscriptionRequests {
       return null;
     }
   }
+
+  public async getSubscriptionByUrn(urnNumber: string): Promise<CaseSubscription> {
```

Comment:
these are getting subscriptions by urn, theyre getting hearings by urn, so this would belong in `hearingsRequests` and it returns a list of hearings too, not caseSubscriptions